dev/service-registry/client-core/src/main/scala/com/lightbend/lagom/internal/registry/package.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2016-2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package com.lightbend.lagom.internal
+
+import java.net.URI
+
+import com.typesafe.config.Config
+
+import scala.collection.immutable
+
+package object registry {
+  private[lagom] def playServerUris(config: Config): immutable.Seq[URI] = {
```

Comment:
s/playServerUris/serviceDnsRecords (or something similar)

The implementation takes the actual values from `play.server.config` in `Config` but we could eventually inject the values using other means. Also, these `URI`s are sometimes ideintifying play services and sometimes lagom services. I think using `playXxxYyy` is confusing.

Reply:
WFM!